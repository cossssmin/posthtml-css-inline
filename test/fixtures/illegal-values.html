<!DOCTYPE html>
<html>
<head>
  <style>
    p {
      font-style: italic !important;
      width: {{ width }};
    }
  </style>
</head>
<body>
  <p>Content</p>
</body>
</html>
