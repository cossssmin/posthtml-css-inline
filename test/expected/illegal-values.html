<!DOCTYPE html>
<html>
<head>
  <style>
    p {
      font-style: italic !important;
      width: {{ width }};
    }
  </style>
</head>
<body>
  <p style="font-style: italic; width: {{ width }};">Content</p>
</body>
</html>
